app/assets/stylesheets/application.css

```diff
@@ -1547,6 +1558,55 @@ input[type="submit"].link_post.pushover_button {
 	padding: 2px;
 }
 
+.radio-group-container {
+	border: 0;
+	padding: 0;
+	margin: 1rem 0 0.5rem;
+
+	legend {
+		float: left;
+		width: 7em;
+		padding: 0;
+	}
+
+	input[type=radio] {
+		margin:0 0.25rem 0 0;
+		vertical-align: text-bottom;
+	}
+
+	.radio-group {
+		display: flex;
+		flex-direction: row;
+		gap: 1rem;
+	}
+
+	/* Used in the user settings page */
+	&.grid-fieldset {
+		margin-bottom: 0.75rem;
+
+		legend {
+			width: 10rem;
+			margin-right: 1rem;
+		}
+	}
+
+	@media only screen and (max-width: 480px) {
+		legend {
+			float: none;
+			margin-bottom: 0.5rem;
+		}
+
+		.radio-group {
+			gap: 1rem;
+
+			label {
+				display: inline;
+				float: none;
+				width: auto !important;
```

Comment:
`!important` is not a shortcut I want to take here, we don't have that much css.